apis/python/src/tiledbsoma/pytiledbsoma.cc

```diff
@@ -68,7 +68,9 @@ PYBIND11_MODULE(pytiledbsoma, m) {
     py::enum_<ResultOrder>(m, "ResultOrder")
         .value("automatic", ResultOrder::automatic)
         .value("rowmajor", ResultOrder::rowmajor)
-        .value("colmajor", ResultOrder::colmajor);
+        .value("colmajor", ResultOrder::colmajor)
```

Comment:
`rowmajor` is already there instead of `row_major` before this PR, so maybe we're 'stuck' with this pattern (antipattern, I think) -- maybe 'too late to change' -- it looks like you're just extending an existing pattern -- but -- throughout core these are called `row_major`, `col_major`, `global_order`, and leaving off the `_` here is, I think, error-prone and not ideal.